server/methods/search/requestMovie.js

```diff
@@ -62,7 +62,22 @@ Meteor.methods({
 			}
 		}
 
-		if (settings.movieApproval) {
+        
+		function approvedUser(user) {
```

Comment:
Was giving this some thought, should we make a new mongo collection for plex users and then use corresponding methods?